### ISSUE TITLE
Error in pod when deploying minio in Rancher

### ISSUE THREAD

**Issue by johnwc** [bitnami/charts#10258]
### Name and Version

bitnami/minio 11.3.5

### What steps will reproduce the bug?

1. Add repo to cluster.
2. Install minio app from charts list

### Are you using any custom parameters or values?

values.yaml
```yaml
gateway:
  auth:
    azure:
      accessKey: '<randomGenKey>'
      secretKey: '<randomGenKey>'
      serviceEndpoint: '<storagename>.core.windows.net'
      storageAccountKey: '<storagekey>'
      storageAccountName: '<storagename>'
  enabled: true
  type: azure
service:
  nodePorts:
    api: 9000
    console: 9001
```

### What is the expected behavior?

Working deployed minio configured in gateway mode using azure storage.

### What do you see instead?

Pod log gives `‘azure’ is not a minio sub-command. See ‘minio --help’.`

**Comment by migruiz4** [comment]
Hi @johnwc,

Thank you for reporting this issue.

MinIO has deprecated the Azure gateway (in addition to GCS and HDFS): https://github.com/minio/minio/pull/14418 
Here you can find the official blogpost about this decision: https://blog.min.io/deprecation-of-the-minio-gateway/

**Comment by migruiz4** [comment]
In order to use the Azure gateway future, you will need to use an older version of the bitnami/minio image.

We will now discuss internally how will we proceed regarding this upstream change.

**Comment by johnwc** [comment]
What version has it still? That blog post contradicts itself, it was posted in February buy it says it will be removed in six months... that would be August.

**Comment by migruiz4** [comment]
According to MinIO changelog, Azure gateway was removed on release `2022-04-30` and GCS/HDFS was removed on `2022-04-26`: https://github.com/minio/minio/releases

So depending on your needs, you have available `bitnami/minio:2022.4.29-debian-10-r0` or `bitnami/minio:2022.4.16-debian-10-r3`

```
$ docker run --rm -it bitnami/minio:2022.4.29-debian-10-r0 bash 
I have no name!@1d8cd73ffb99:/opt/bitnami/minio-client$ minio gateway --help
NAME:
  minio gateway - start object storage gateway

USAGE:
  minio gateway COMMAND [COMMAND FLAGS | -h] [ARGUMENTS...]

COMMANDS:
  nas    Network-attached storage (NAS)
  azure  Microsoft Azure Blob Storage
  s3     Amazon Simple Storage Service (S3)
```

```
$ docker run --rm -it bitnami/minio:2022.4.16-debian-10-r3 bash
I have no name!@1c68c133e5fa:/opt/bitnami/minio-client$ minio gateway --help
NAME:
  minio gateway - start object storage gateway

USAGE:
  minio gateway COMMAND [COMMAND FLAGS | -h] [ARGUMENTS...]

COMMANDS:
  nas    Network-attached storage (NAS)
  azure  Microsoft Azure Blob Storage
  s3     Amazon Simple Storage Service (S3)
  hdfs   Hadoop Distributed File System (HDFS)
  gcs    Google Cloud Storage
```

**Comment by corico44** [comment]
Finally, all gateway features have been removed from Minio. The related options have been removed in version 12.1.0.
